cmd/neofs-adm/internal/modules/morph/balance.go

```diff
@@ -126,17 +124,17 @@ func dumpBalances(cmd *cobra.Command, _ []string) error {
 	if dumpAlphabet {
 		alphaList := make([]accBalancePair, len(irList))
 
-		w := io.NewBufBinWriter()
+		b := smartcontract.NewBuilder()
 		for i := range alphaList {
-			emit.AppCall(w.BinWriter, nnsCs.Hash, "resolve", callflag.ReadOnly,
-				getAlphabetNNSDomain(i),
-				int64(nns.TXT))
+			b.InvokeMethod(nnsCs.Hash, "resolve", getAlphabetNNSDomain(i), int64(nns.TXT))
```

Comment:
BTW, builder invokes this with `callflag.All`, and it was ReadOnly before. But anyway, it's our contract and there's no harm in it.

Reply:
oh, yeah, forgot to ping you about that. @roman-khimov, yes, in some cases there were readonly calls for readonly operations but nothing should break because of it. tbh, i really do not see much difference b/w `InvokeMethod` and `AppCall` except the call flags (and sometimes we need them)

Reply:
I think it's OK to have this RO -> ALL replacement in Adm.

Reply:
I've seen this, it doesn't matter much for these calls, most of them are test invocations only.